app/views/admin/facility_groups/_form.html.erb

```diff
@@ -37,6 +37,8 @@
         <%= form.select(:protocol_id, protocols.order(:name).map { |protocol| [protocol.name, protocol.id] }) %>
       <% end %>
 
+      <%= form.number_field :district_estimated_population, placeholder: "0", label: "Estimated hypertensive population" %>
```

Comment:
Are you sure you want a placeholder of 0 here? It may make sense to leave this nil if the user doesn't enter anything, so that we aren't running the sum callback with a population of 0.

Reply:
That's a great point @rsanheim — I'll keep the population field blank by default.